test/blackbox-tests/test-cases/auto_open.t

```diff
@@ -0,0 +1,31 @@
+auto_open is a library field used to select which modules are going to be open
+by default:
+
+  $ cat >dune-project <<EOF
+  > (lang dune 3.11)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (library
+  >  (name foo)
+  >  (wrapped false)
+  >  (modules foo bar)
+  >  (auto_open Foo Bar))
```

Comment:
Does it matter that `Foo` is captilized here? Reading the code, it seems like it can be both right?

Reply:
Yeah, it can be both.